.circleci/config.yml

```
@@ -0,0 +1,198 @@
+            e=1 && for i in {1..5}; do
+                docker build --rm --cache-from=nitransforms:latest \
+                -t nitransforms:latest \
+                --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
+                --build-arg VCS_REF=`git rev-parse --short HEAD` . \
+                && e=0 && break || sleep 15
+            done && [ "$e" -eq "0" ]
+            docker tag nitransforms:latest localhost:5000/nitransforms
+            docker push localhost:5000/nitransforms
+    - run:
+        name: Docker registry garbage collection
+        command: |
+            docker exec -it registry /bin/registry garbage-collect --delete-untagged \
+                /etc/docker/registry/config.yml
+    - save_cache:
+        key: build-v1-{{ .Branch }}-{{ epoch }}
+        paths:
+          - /tmp/docker
+    - run:
+        name: Store FreeSurfer license file
+        command: |
+            mkdir -p /tmp/fslicense
+            cd /tmp/fslicense
+            echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW12RVYzelRmZ1xuRlM1Si8yYzFhZ2c0RVxuIiA+IGxpY2Vuc2UudHh0Cg==" | base64 -d | sh
+    - persist_to_workspace:
+        root: /tmp
+        paths:
+          - fslicense
+
+  test_pytest:
+    machine:
+      image: circleci/classic:201808-01
+    environment:
+      FS_LICENSE: "/tmp/fslicense/license.txt"
+    working_directory: /tmp/tests
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - checkout:
+          path: /tmp/src/nitransforms
+      - run:
+          name: Get codecov
+          command: python -m pip install codecov
+      - restore_cache:
+          keys:
+            - build-v1-{{ .Branch }}-{{ epoch }}
+            - build-v1-{{ .Branch }}-
+            - build-v1-master-
+            - build-v1-
+      - run:
+          name: Set-up a Docker registry & pull
+          command: |
+            docker run -d -p 5000:5000 --restart=always --name=registry \
+                -v /tmp/docker:/var/lib/registry registry:2
+            docker pull localhost:5000/nitransforms
+            docker tag localhost:5000/nitransforms nitransforms:latest
+      - run:
+          name: Run unit tests
+          no_output_timeout: 2h
+          command: |
+            mkdir -p $PWD/artifacts $PWD/summaries
+            sudo setfacl -d -m group:ubuntu:rwx $PWD
+            sudo setfacl -m group:ubuntu:rwx $PWD
+            docker run -u $( id -u ) -it --rm=false -w /src/nitransforms \
+              -e COVERAGE_FILE=/tmp/summaries/.pytest.coverage \
+              -v /tmp/fslicense/license.txt:/opt/freesurfer/license.txt:ro \
+              -v ${PWD}:/tmp nitransforms:latest \
+              pytest --junit-xml=/tmp/summaries/pytest.xml \
+                     --cov nitransforms --cov-report xml:/tmp/summaries/unittests.xml \
+                     --ignore=nitransforms/tests/ \
+                     --ignore=nitransforms/joss/ \
+                     --ignore=nitransforms/docker/ \
```

Since the working directory has been set to `/src/nitransforms`, at least the last two folders are unnecessary.
```suggestion
                     --ignore=nitransforms/tests/ \
```

I believe you actually don't want to ignore `nitransforms/tests/` here. If that is the case then:
```suggestion
```